README.md

```diff
@@ -1,3 +1,5 @@
+[![Puppet Forge](http://img.shields.io/puppetforge/v/katello/gutterball.svg)](https://forge.puppetlabs.com/katello/gutterball)
```

Comment:
It wasn't loading for me, but that might have been a temporary service issue rather than http/https. Carry on :)